pom.xml

```diff
@@ -474,6 +474,12 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- MODEL MAPPER -->
+        <dependency>
+            <groupId>org.modelmapper</groupId>
+            <artifactId>modelmapper</artifactId>
+            <version>0.7.7</version>
+        </dependency>
```

Comment:
There are some problems with this model mapper. And I`ve removed it from the project